audit-connector/src/main/java/com/sequenceiq/cloudbreak/audit/util/ActorUtil.java

```diff
@@ -1,23 +1,27 @@
 package com.sequenceiq.cloudbreak.audit.util;
 
-import static com.sequenceiq.cloudbreak.auth.ThreadBasedUserCrnProvider.INTERNAL_ACTOR_CRN;
+import javax.inject.Inject;
 
 import org.springframework.stereotype.Component;
 
 import com.sequenceiq.cloudbreak.audit.model.ActorBase;
 import com.sequenceiq.cloudbreak.audit.model.ActorCrn;
 import com.sequenceiq.cloudbreak.audit.model.ActorService;
+import com.sequenceiq.cloudbreak.auth.crn.RegionalAwareInternalCrnGeneratorFactory;
 
 @Component
 public class ActorUtil {
 
+    @Inject
+    private RegionalAwareInternalCrnGeneratorFactory regionalAwareInternalCrnGeneratorFactory;
```

Comment:
nit: I think the name should be "RegionAware..." not "RegionalAware..."

Reply:
the simple crn generation is also in RegionAwareCrnGenerator so then we should rename that as well right?